org.eclipse.scanning.event/src/org/eclipse/scanning/event/AbstractQueueConnection.java

```diff
@@ -478,17 +484,17 @@ public boolean replace(U bean, String queueName) throws EventException {
 				}
 			}
 			if (!found) throw new EventException("Cannot find bean '"+bean.getName()+"' in submission queue!\nIt might be running now.");
-			
+
 			clearQueue(queueName);
-			
-			ISubmitter<U> submitter = this instanceof ISubmitter 
-					                ? (ISubmitter<U>)this 
+
+			ISubmitter<U> submitter = this instanceof ISubmitter
+					                ? (ISubmitter<U>)this
 					                : (ISubmitter<U>)eservice.createSubmitter(getUri(), queueName);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "ISubmitter". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)